metatron-app/metatron-discovery
# fix datasource information

### Description

fix datasource information
fix datasource validation

**Related Issue** : 

METATRON-2842
METATRON-2854
METATRON-2879
METATRON-2895
METATRON-2862
METATRON-2930
METATRON-2911
METATRON-2919

### How Has This Been Tested?

METATRON-2842
1. Create a datasource
2. Add a column with a duplicate name.
3. See the error message

METATRON-2854
1. Create a datasource
2. Select a column as the timestamp column.
3. Select next button with no input
4. See the error message (Required)

METATRON-2879
1. Create a datasource with a datasource
2. Step4 Ingestion settings > Batch cycle
3. Select User defined
4. If it is a valid value, "Valid crontab" is shown.
5. If you click the Next button without checking the validation, "Check up validation" is shown.

METATRON-2895
1. Create a datasource
2. Check the title (AS-IS Configure schema / TO-BE Please configure the schema)

METATRON-2862
1. Go to the data connection list
2. Check the placeholder in the upper right search box with Korean language.
3. AS-IS 데이터커넥션 이름으로 검 / TO-BE 데이터커넥션 이름으로 검색

METATRON-2930
1. Create a datasource with the excel file.
2. After creating the data source, the sheet name is shown in the information tab.

METATRON-2911
1. Go to the datasource detail.
2. Check the histogram detail
3. Data Range: Display the actual time value loaded in the engine (Display the time range reflecting the Segment Granularity unit)
4. Histogram X-axis: Display Time Range Reflecting Segment Granularity Unit
5. Histogram tooltips: display row counts, size

METATRON-2919
1. Go to the datasource detail.
2. Check the Physical name.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
